src/etc/inc/plugins.inc.d/dhcpd.inc

```diff
@@ -1826,9 +1827,10 @@ function dhcpd_dhcrelay6_configure($verbose = false)
         }
 
         if (!isset($destif)) {
-            if (is_array($config['gateways']['gateway_item'])) {
-                foreach ($config['gateways']['gateway_item'] as $gateway) {
-                    if (isset($gateway['defaultgw'])) {
+            $gateways = iterator_to_array((new \OPNsense\Gateways\Gateway)->gatewayIterator());
+            if (is_array($gateways)) {
```

Comment:
I'm pretty sure this is always an array now and safe to feed to foreach directly